### PR TITLE
Fix a bug of combineContainedIn for RecognizePlaces.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: internal/server/recon/golden/recognize_places/result.json [hunk 1]
```diff
@@ -69,6 +69,38 @@
...
+      "items": [
+        {
+          "span": "I went to"
+        },
+        {
+          "span": "Half Moon Bay California",
+          "places": [
+            {
+              "dcid": "geoId/06"
+            },
```
Comment (pradh) [inline internal/server/recon/golden/recognize_places/result.json:82]: Why does this match california?

File: internal/server/recon/golden/recognize_places/result.json [hunk 2]
```diff
@@ -69,6 +69,38 @@
...
+        },
+        {
+          "span": "and"
+        },
+        {
+          "span": "California , Washington County",
+          "places": [
+            {
+              "dcid": "geoId/42125"
+            },
```
Comment (pradh) [inline internal/server/recon/golden/recognize_places/result.json:96]: ditto, why does it match county?

If this is a combined span, it should be only geoId/4210768, right?

File: internal/server/recon/golden/recognize_places/result.json (hunk 1 above)
Comment (Spaceenter) [reply internal/server/recon/golden/recognize_places/result.json]: Changed

File: internal/server/recon/golden/recognize_places/result.json (hunk 2 above)
Comment (Spaceenter) [reply internal/server/recon/golden/recognize_places/result.json]: Done

**Review by pradh — APPROVED** [review]
Thanks Wei!

**Pull request merged**